ci-operator/step-registry/ipi/install/hosted-loki-staging/ipi-install-hosted-loki-staging-commands.sh

```diff
@@ -6,8 +6,8 @@ set -o pipefail
 
 export LOKI_STAGE_ENDPOINT=https://observatorium.api.stage.openshift.com/api/logs/v1/dptp/loki/api/v1
 export LOKI_PROD_ENDPOINT=https://observatorium.api.openshift.com/api/logs/v1/dptp/loki/api/v1
-export PROMTAIL_IMAGE="quay.io/openshift-logging/promtail"
```

Comment:
I'd rather work with "openshift-logging" to have multiarch version uploaded

Reply:
That's currently not an option, but we'll keep it in mind as we continue to work on enablement.